### DISCUSSION TITLE
Icon map

### DISCUSSION THREAD

**Discussion by wildsonc** [apache/superset#18314] [Q&A / Help]
Is there any way to change the map icon?
![image](https://user-images.githubusercontent.com/72408418/126224892-3c9c6d09-5373-4b58-9a29-0d3121cd0c07.png)

**Comment by nytai** [comment]
Do you mean the car? Not sure how you got that, can you provide steps to repro with some of the example data sets?

**Comment by wildsonc** [comment]
I did it in photoshop, I wanted to know if I can change the icon in the superset itself, such as passing an image url in js data mutator for example.

**Comment by junlincc** [comment]
that sounds cool!  @rusackas @michael-s-molina @geido @pkdotson if you guys know, please let the community know :)

**Comment by wildsonc** [comment]
I used [IconLayer](https://deck.gl/docs/api-reference/layers/icon-layer) also from Deck.gl, and the result was very good

![image](https://user-images.githubusercontent.com/72408418/126816351-eebccad7-6e48-45a5-823e-8c0f7d07abd3.png)

**Reply by dtaytson** [reply]
Hi, wildsonc.
Please, could you explain how u did it ?

**Reply by Gabrielolliverl** [reply]
Please, could you explain how u did it ?

**Reply by rusackas** [reply]
I'm interested too! I haven't seen that trick before.
